lib/themes/dosomething/paraneue_dosomething/scss/patterns/_counter.scss

```diff
@@ -1,12 +1,35 @@
 // @TODO: Move to Forge if this is shipped.
 .counter {
-  display: inline-block;
   color: $med-gray;
   font-weight: $weight-bold;
-  font-size: 13px !important;
-  padding-left: 3px;
+  font-size: 16px;
+  padding-left: 4px;
 
   &.is-active {
-    color: $purple;
+    color: $dark-gray;
+  }
+}
+
+.photo__actions li {
+  * {
+    display: inline;
+    vertical-align: middle;
+  }
+
+  img {
+    width: 30px !important;
```

Comment:
Added a custom class but this rule is still overriding the 30px width. I'll track down where the .photo.-framed img is coming from...